frontend/src/plugins/router/middlewares/login.ts

```diff
@@ -41,7 +41,11 @@ export const loginGuard = async (
 ): Promise<Exclude<NavigationGuardReturn, Error>> => {
   const toServerPages = serverPages.has(to.name);
 
-  if (!jsonConfig.allowServerSelection && toServerPages) {
+  /**
+   * Do not allow the server selection pages if allowServerSelection=false,
+   * but do allow the login page.
+   */
+  if (!jsonConfig.allowServerSelection && toServerPages && to.name != '/server/login') {
```

Comment:
```suggestion
  /**
   * Do not allow the server selection pages if `allowServerSelection` is false in config.json,
   * but do allow the login page.
   */
  if (!jsonConfig.allowServerSelection && (toServerPages && to.name !== serverLoginUrl)) {
```

Please run ESLint before pushing to verify new code does not introduce new problems. This was being flagged by the ``eqeqeq`` rule. Also:
* The login url was defined as a constant in the top of the file and we were repeating it here unnecessarily
* Added parenthesis to clarify the different sections of the conditional for readability.
* If we're adding comments here (the logic is simple enough to be understandable and I don't believe it required comments in the first place) but if we're doing them anyway, let's them be top notch! 👌. I believe explaining that `allowServerSelection` comes from the config file (although it's implicit if someone tracks the source of the `jsonConfig` variable where further comments are located).

Reply:
I agree with all of this! It's been a while so I expected some styling edits :)